mne/report/report.py

```diff
@@ -1660,7 +1768,6 @@ def _add_ica_overlay(self, *, ica, inst, image_format, section, tags, replace):
 
         fig = ica.plot_overlay(inst=inst_, show=False, on_baseline="reapply")
         del inst_
-        _constrain_fig_resolution(fig, max_width=MAX_IMG_WIDTH, max_res=MAX_IMG_RES)
```

Comment:
DRYed the code by making `_fig_to_img` always do the constraining rather than calling this all over the place